### PR TITLE
Closes #2 ECI-10 - added deeplink to Fieldwire in Project Details view

### LINKED ISSUE #2: Project Details View: Add deeplink to Fieldwire project
In the Project Details View let us generate the URL and give the user an easy way to jump into the Fieldwire project. <br><br>Format: [https://app.fieldwire.com/#!/projects/55264066-eec0-468d-bb87-45fb17de1621/plans?lang=en](https://app.fieldwire.com/#!/projects/55264066-eec0-468d-bb87-45fb17de1621/plans?lang=en)

### PR DESCRIPTION
**Pull request by martin-mcgloin**
Just added a deeplink, had to install chackra/icons to support the button you wanted @anaviz 

Closes Linear issue: ECI-10

### CODE REVIEW

File: src/pages/ProjectDetails.jsx [hunk 1]
```diff
@@ -141,11 +142,16 @@ const ProjectDetails = () => {
...
               <Text marginBottom="24px">{fwProjectData.fw_project_id}</Text>
-
               <Text fontWeight="bold" marginBottom="4px">
                 Number of Tasks
               </Text>
               <Text marginBottom="24px">{fwProjectData.num_tasks}</Text>
+              <Text fontWeight="bold" marginBottom="4px">
+                Fieldwire Project Link
+              </Text>
+              <Button as="a" href={`https://app.fieldwire.com/#!/projects/${fwProjectData.fw_project_id}/plans?lang=en`} target="_blank" rel="noopener noreferrer" rightIcon={<ArrowForwardIcon />} colorScheme="teal" variant="outline">
```
Comment (anaviz) [inline src/pages/ProjectDetails.jsx:152]: The fieldwire url We can probably use a constant for the fieldwire url doesn't need "?lang=en"

Comment (martin-mcgloin) [reply src/pages/ProjectDetails.jsx:152]: Agree, let us not hardcoded the URL - shall we make a constants file?

**Comment by martin-mcgloin** [comment]
@anaviz - is this better? Good feedback!

**Comment by anaviz** [comment]
All good M @martin-mcgloin

**Pull request merged**